lib/compilers/tinyc.js

```diff
@@ -36,7 +36,7 @@ export class TinyCCompiler extends BaseCompiler {
             if (!_.some(userOptions, (opt) => opt === '-E')) {
                 filters.binary = true;
             }
-            return ['-o', this.filename(outputFilename)];
+            return ['-g', '-o', this.filename(outputFilename)];
```

Comment:
Should maybe be a separate change (or did you say that in the PR?)

Reply:
Yeah, I screwed my git-fu and had to push a few unreleated changes as part of the pr, sorry :(